modules/components/src/Query.js

```diff
@@ -5,7 +5,11 @@ import defaultApi from './utils/api';
 import { defaultProps } from 'recompose';
 
 class Query extends Component {
-  state = { data: null, error: null, loading: false };
+  constructor(props) {
+    super(props);
+    const { shouldFetch } = props;
+    this.state = { data: null, error: null, loading: shouldFetch && true };
+  }
```

Comment:
since `shouldFetch` is a bool I don't think we need the `&&`. also I think `this.props` is available as follows:
```
class Query extends Component {
  state = { data: null, error: null, loading: this.props.shouldFetch };
  ...
}
```